examples/api/src/app/api/open-graph/lib/util.ts

```diff
@@ -35,15 +35,18 @@ async function fetchUserData(address: string): Promise<FarcasterUser | null> {
 }
 
 export function toUrlMetadata(nftMetadata: NFTMetadata): UrlMetadata {
+  // Assume that creator is a FarcasterUser
```

Comment:
what happens when they're not

Reply:
in the opengraph api context it will always be, but have added checks to be sure since it's an exported function